listener_sub: # Name of the node
  ros__parameters:
    aruco_marker_0: "right_90"
    aruco_marker_1: "left_90"
    aruco_marker_2: "end"
